libminifi/src/utils/tls/CertificateUtils.cpp

```diff
@@ -118,7 +118,7 @@ EVP_PKEY_unique_ptr extractPrivateKey(const PCCERT_CONTEXT certificate) {
       auto const blob = reinterpret_cast<BCRYPT_RSAKEY_BLOB *>(data.get());
 
       if (blob->Magic == BCRYPT_RSAFULLPRIVATE_MAGIC) {
-        auto rsa = RSA_new();
+        auto rsa = RSA_new();  // 'RSA_new': deprecated since OpenSSL 3.0
```

Comment:
Is there a separate Jira ticket to address these deprecated OpenSSL calls?

Reply:
there is one now :)

https://issues.apache.org/jira/browse/MINIFICPP-2191